src/affexpr.jl

```diff
@@ -181,10 +181,10 @@ function addconstraint(m::Model, c::LinearConstraint)
     end
     return LinConstrRef(m,length(m.linconstr))
 end
-addconstraint(m::Model, c::Array{LinearConstraint}) =
+addconstraint(m::Model, c::AbstractArray{LinearConstraint}) =
     error("The operators <=, >=, and == can only be used to specify scalar constraints. If you are trying to add a vectorized constraint, use the element-wise dot comparison operators (.<=, .>=, or .==) instead")
 
-function addVectorizedConstraint(m::Model, v::Array{LinearConstraint})
+function addVectorizedConstraint(m::Model, v::AbstractArray{LinearConstraint})
```

Comment:
This is an internal function, it's unclear how it could be called with an ``AbstractArray``